src/main/java/org/takes/rq/RqHref.java

```diff
@@ -72,19 +76,22 @@ public Href href() throws IOException {
                 .header("host").iterator();
             final Iterator<String> protos = new RqHeaders.Base(this)
                 .header("x-forwarded-proto").iterator();
-            final String host;
+            Text host = new TextOf("localhost");
```

Comment:
@fanifieiev Remember, all variables should be final

Reply:
@paulodamaso I think it should be checked by PMD right?